services/movie.py

```diff
@@ -0,0 +1,38 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+from db.models import Genre
+from db.models import Actor
+
+
+def get_movies(
+        genres_ids: list[Genre.pk] = None,
+        actors_ids: list[Actor.pk] = None

```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contain string elements and can be None has typehint Optional[list[str]]

Genre.pk is an int value, swap it